src/igvfd/tests/test_types_biosample.py

```diff
@@ -36,10 +36,94 @@ def test_age_calculation(testapp, in_vitro_cell_line):
         in_vitro_cell_line['@id'],
         {'lower_bound_age': 90, 'upper_bound_age': 90, 'age_units': 'year'})
     res = testapp.get(in_vitro_cell_line['@id'])
-    print(res.json.get('age'))
     assert res.json.get('age') == '90 or above'
     res = testapp.patch_json(
         in_vitro_cell_line['@id'],
         {'age_units': 'month'})
     res = testapp.get(in_vitro_cell_line['@id'])
     assert res.json.get('age') == '90'
+
+
+def test_summary(testapp, tissue, primary_cell, whole_organism, in_vitro_cell_line, technical_sample, sample_term_endothelial_cell, sample_term_embryoid_body, sample_term_lymphoblastoid):
```

Comment:
Can we add a test here for the case where 'tissue' is in the term name and in the biosample type or classification?